aio/deploy/helm-chart/kubernetes-dashboard/templates/deployment.yaml

```diff
@@ -143,9 +143,13 @@ spec:
         volumeMounts:
         - mountPath: /tmp
           name: tmp-volume
-{{- with .Values.metricsScraper.containerSecurityContext }}
+
+{{- if .Values.metricsScraper.containerSecurityContext }}
```

Comment:
Isn't it a bit weird that dashboard deployment refers to the `metricsScraper.containerSecurityContext`? It should be independent.

Reply:
Sorry, I don't know exactly what you mean. This if value checks if there are separate containerSecurityContext values set for the metrics scraper. When yes, they should take place, if not the else if will check if there are containerSecurityContext globally set. This should guarantee, that people who set individual containerSecurityContext values for the metrics scraper cannot do that after upgrading to this helm chart version. 
Or do you mean there is a simpler way to achieve this behaviour in code?

Reply:
Sorry, I was under the impression that this is part of the "pure" kubernetes-dashboard deployment. Now that I am looking more closely I see that it is a single deployment with multiple containers. I had no idea our helm deployment differs this much from our `recommended.yaml` deployment :slightly_smiling_face: 

I think this is the best solution then.